packages/ui/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "ui",
+  "version": "0.1.0",
+  "description": "Avocado UI",
+  "keywords": [
+    "whizzes",
+    "avocado",
+    "ui"
+  ],
+  "author": "whizzes (https://github.com/whizzes)",
+  "homepage": "https://github.com/whizzes/avocado#readme",
+  "license": "MIT",
+  "main": "lib/ui.js",
+  "directories": {
+    "src": "src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/whizzes/avocado.git"
+  },
+  "scripts": {
+    "start": "webpack-dev-server  --mode development --hot",
+    "build": "webpack --mode production",
+    "dist": "npm run build && mkdir -p ../desktop/build/ui && cp -R ./build/** ../desktop/build/ui && rm -rf ./build/",
+    "lint": "eslint --fix \"**/*.{ts,tsx}\""
```

Comment:
This command should be
`
{ "lint": "eslint \"**/*.{ts,tsx}\"" }
`

And add another command:
`
{"lint:fix": "eslint --fix \"**/*.{ts,tsx}\""}
`